config/features.yml

```diff
@@ -365,6 +365,14 @@ features:
     actor_type: user
     description: Uses local_bgs rather than bgs-ext
     enable_in_development: true
+  claims_api_v1_lh_fes_auto_establish_claim_enabled:
+    actor_type: user
+    description: Use Form526 Establishment Service (FES) for v1 disability compensation claims auto-establishment
+    enable_in_development: true
+  claims_api_v2_lh_fes_auto_establish_claim_enabled:
```

Comment:
Looks like Jayson added the required feature in a previous PR: lighthouse_claims_api_v2_enable_FES
We can remove this one and update the spots in code to use the one already added.